scripts/notes

```diff
@@ -8,18 +8,6 @@ source "$rvm_scripts_path/functions/logging"
 source "$rvm_scripts_path/functions/support"
 source "$rvm_scripts_path/functions/utility"
 
-rvm_warn "\
-In case of problems: ${rvm_notify_clr:-}https://rvm.io/help ${rvm_warn_clr:-}and ${rvm_notify_clr:-}https://twitter.com/rvm_io"
```

Comment:
maybe something like:

    In case of problems open a ticket: https://github.com/rvm/rvm/issues

it does not have to be obvious to everybody